Makefile

```diff
@@ -1,4 +1,7 @@
 artifact_name       := company-lookup.web.ch.gov.uk
+commit              := $(shell git rev-parse --short HEAD)
+tag                 := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version             := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
```

Comment:
The previous versioning scheme is being deprecated in favour of automatic versioning of build artefacts, so we should no longer attempt to determine the version based on commit or tag. Instead, you could replace these lines with for example:

```
version := unversioned
```

This way, when running `make bulid` manually the `.jar` file will be packaged under `target/` with `unversioned` in the name before being copied to the parent directory and stripped of the `unversioned` tag.

When running under Concourse CI the value of the `version/version` file (if any) would replace the `unversioned` value of this variable meaning the conditional `ifndef` step in the `package` target works as expected (if `version/version` does not exist then `version` will be an empty string and fail the conditional check).